src/lib/modernExtend.ts

```diff
@@ -842,3 +856,57 @@ export function occupancy(args?: Partial<BinaryArgs>): ModernExtend {
 
     return result;
 }
+
+export function ignoreClusterReport(args: {cluster: string | number}): ModernExtend {
+    const fromZigbee: Fz.Converter[] = [{
+        cluster: args.cluster.toString(),
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {},
+    }];
+
+    return {fromZigbee, isModernExtend: true};
+}
+
+export function iasGas(args: {variant: 'alarm_1' | 'alarm_2'}): ModernExtend {
```

Comment:
I propose to name this `iasAlarm` and add an additional arg to specify the property (e.g. `contact`, `gas`)

Reply:
That will be a big extend, but sounds reasonable enough to encapsulate everything iasZone related